manifests/openshift-csi-resizer-storageclass-reader-role.yaml

```diff
@@ -0,0 +1,13 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: openshift-csi-resizer-storageclass-reader-role
```

Comment:
This role is suspicious - is there a resizer that does not need to read StorageClasses?